app/views/planning_applications/assessment/assessment_details/neighbour_summary/new.html.erb

```diff
@@ -69,7 +69,7 @@
               :"#{tag}",
               label: {text: "Summary of #{tag.to_s.humanize.downcase} comments", size: "m"},
               rows: 10,
-              value: (@assessment_detail.entry[/(?<=#{tag.to_s.humanize}:)(.*)/] unless @assessment_detail.entry.nil?)
+              value: @rejected_assessment_detail&.entry&.[](/(?<=#{tag.to_s.humanize}:)(.*)/)
```

Comment:
this line isn't super obvious to me 

first off, is it intentional that we've gone from `[/regex/]` to `[](/regex/)`? I tested briefly and this doesn't seem to work but I might be missing something. adding the `&` seems also to be invalid syntax.

secondly, i'm guessing this is meant to extract the part of the entry that follows the tagged name and a colon. but i'm not a fan of interpolating a variable into the regex. would it actually work to just use `entry&.split("#{tag.to_s.humanize}:")[1]` for example?

finally maybe extracting this line into a variable would help readability